src/Lib/package.cpp

```diff
@@ -34,17 +40,61 @@ const std::string Package::GetFullyQualifiedName(void) {
   return Package::GetName() + " " + Package::GetVersion();
 }
 
+const void Package::assignDataDir(void) {
+  const char *possible[] = {".", "..", DATADIR, NULL};
```

Comment:
I think `DATADIR` should be checked first incase it's overridden.

Reply:
I moved it, but wanted to suggest checking first for the environmental variable "$NETPANZER_DATADIR".